File: PRAKTIKUM_3/PRAK305-2410817210004-Rahmad.py
import math
a=int(input("masukan waktu dalam format detik :"))
if a>=86400 :
    hari = a//86400
    jam=(a%86400)//3600
    menit=((a%86400)%3600)//60
    detik =a%60
    print(f"{hari:.0f} hari : {jam:.0f} jam : {menit:.0f} menit : {detik:.0f} : detik")
else :
    jam= a//3600
    menit=(a%3600)//60
    detik =a%60
    print(f"{jam:.0f} jam : {menit} menit : {detik:.0f} : detik")
 

File: PRAKTIKUM_3/PRAK305-2410817210004-Rahmad.c
#include <stdio.h>

int main() {
    int a;
    
    printf("Masukkan waktu dalam format detik : ");
    scanf("%d", &a);
    if(a>=86400){
        int hari = a/86400;
         int jam=(a%86400)/3600, menit=((a%86400)%3600)/60, detik =a%60;
        printf("%d hari : %d jam : %d menit : %d detik",hari,jam,menit,detik);      
        }
    else {
         int jam=a/3600, menit=(a%3600)/60, detik =a%6;
         printf("%d jam : %d menit : %d detik ",jam,menit,detik);
    }
} 